common/religion/doctrines/00_core_tenets.txt

```diff
@@ -2529,5 +2820,207 @@
 	#		sins = { craven = 2}
 	#	}
 	#}
+	
+	##############################
+	#	Warcraft
+	##############################
```

Comment:
I suggest creating our own file with tenets instead

Reply:
I tried, it didn't work. The game has seen them as separate tenets.